security/cmd/istio_ca/main.go

```diff
@@ -189,9 +187,6 @@ func init() {
 	flags.IntVar(&opts.grpcPort, "grpc-port", 8060, "The port number for Citadel GRPC server. "+
 		"If unspecified, Citadel will not serve GRPC requests.")
 
-	flags.StringVar(&opts.grpcHostname, "grpc-hostname", "istio-ca", "deprecated")
```

Comment:
General rule is to remove the uses of a flag in one PR, and remove the actual flag in separate one ( in next release ).

Was grpc-hostname present in 1.0 ? If not - probably not a problem.